tests/phpunit/elementor/core/files/css/test-base.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Files\Css;
+
+use ElementorEditorTesting\Elementor_Test_Base;
+
+/**
+ * Test the CSS Base class
+ */
+class Test_Base extends Elementor_Test_Base {
+
+	public function test_parse_property_placeholder() {
```

Comment:
This test should test the typical situation when you pass any standard value and not an edge case, so change the name or change the test :)

Reply:
Also add tests for all the edge cases: "any_type_of_empty", "0" and 0 (as you already did).

Reply:
Done